.orchestra/ci/ci.sh

```diff
@@ -148,6 +156,11 @@ for TARGET_COMPONENT in $TARGET_COMPONENTS; do
     fi
 done
 
+if [ "$PUSH_CHANGES" != 1 ]; then
+	echo '$PUSH_CHANGES != 1, exiting without pushing changes'
+	exit $RESULT
+fi
```

Comment:
This early exit introduces the risk that if we append something at the end of the script it accidentally skipped.
Wrap the rest of the code in the `if`.